dashboard/db/migrate/20170814203015_add_share_flag_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddShareFlagToUser < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :sharing_allowed, :boolean, null: false, default: true
```

Comment:
For the comment, we should use the [Rails 5 migration comment](http://blog.bigbinary.com/2016/06/21/rails-5-supports-adding-comments-migrations.html), which will persist in the DB itself and in the schema.rb